<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="sameer" name="testNgPrograms.Actitime_Timeout" tests="1" failures="0" timestamp="1 Mar 2023 06:36:49 GMT" time="0.211" errors="1">
  <testcase name="openBrowserAndAppURL" time="0.211" classname="testNgPrograms.Actitime_Timeout">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.openBrowserAndAppURL() didn&#039;t finish within the time-out 200">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.openBrowserAndAppURL() didn't finish within the time-out 200
	at java.lang.Thread.sleep(Native Method)
	at testNgPrograms.Actitime_Timeout.openBrowserAndAppURL(Actitime_Timeout.java:10)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
]]>
    </error>
  </testcase> <!-- openBrowserAndAppURL -->
</testsuite> <!-- testNgPrograms.Actitime_Timeout -->
